.github/workflows/tests.yml

```diff
@@ -53,6 +53,35 @@ jobs:
         with:
           name: bazel-testlogs
           path: bazel-testlogs
+
+  build_bazel4_arm64_simulator:
+    name: Build arm64 Simulator
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Select Xcode 12.2
+        run: sudo xcode-select -s /Applications/Xcode_12.2.app
+      - name: Build and Test
+        run: |
+          bazelisk clean # Note: there is a problem with some reuse in rules_swift
```

Comment:
What problem? Is there an issue we can link to here?

Reply:
It seems to flake out when cross compiling, fixed in Bazel 5